dockerfiles/incr.sh

```diff
@@ -26,6 +26,9 @@
 
 # NB: INPUT_* may be pre-set as environment variables.
 
+#The name of your "upstream" git remote
+UPSTREAM=upstream
+
 while getopts “d:fhlpt:n:u:b:” opt; do
```

Comment:
new option r not included in getopts call.

Reply:
Fixed